spec/features/admin_claim_approve_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+
+RSpec.feature "Admin approves a claim" do
+  context "User is logged in as a service operator" do
+    before do
+      stub_dfe_sign_in_with_role(AdminSession::SERVICE_OPERATOR_DFE_SIGN_IN_ROLE_CODE)
+      visit admin_path
+      click_on "Sign in"
+    end
+
+    scenario "User can approve a claim" do
+      freeze_time do
+        submitted_claims = create_list(:claim, 5, :submitted, submitted_at: DateTime.now, approved_at: nil)
+        claim_to_approve = submitted_claims.first
+
+        click_on "Approve claims"
+
+        expect(page).to have_content(claim_to_approve.reference)
+
+        find("a[href='#{admin_claim_path(claim_to_approve)}']").click
+        click_on "Approve"
+
+        claim_to_approve.reload
+
+        expect(claim_to_approve.approved_at).to eq(DateTime.now)
+        expect(claim_to_approve.approved_by).to eq("123")
```

Comment:
If we set an actual `user_id` explicitly in the call to `stub_dfe_sign_in_with_role` above, we can assert that this value matches the value we stub, which will make for a more thorough and less brittle spec.